dwave/optimization/generators.py

```diff
@@ -460,7 +460,6 @@ class as the decision variable being optimized, with permutations of its
         sublist representing various itineraries for each vehicle.
     """
 
-    time_distances = np.asarray(time_distances)
```

Comment:
This line is redundant to an eventual `_requires()` call, and in an early implementation of this PR was causing issues. I kept this fix in even though I didn't eventually need it.